app/views/members/activities/partials/_view.html.haml

```diff
@@ -45,6 +45,12 @@
           \-
           = activity.end_time.strftime("%H:%M")
       <br/>
+      - if activity.open_date != nil && activity.open_date > Date.today
+        %strong= I18n.t("activerecord.labels.activities.opens_at")
+        = l(activity.open_date, format: "%A %d/%m/%Y")
+        - if !activity.open_time.nil?
```

Comment:
I'm unsure about the interactivity between ruby and haml but the activity model also contains a `when_open` method which includes both time and date. Current logic suggests to me that the open date will be visible for the whole day the activity opens; is that intended?

Reply:
Good catch, it is not.

Looking into it further, there is actually a function on Activity: `open` (`:: bool`), so I'll just use the inverse of that.